python/mxnet/symbol.py

```diff
@@ -1068,7 +1172,24 @@ def simple_bind(self, ctx,
 
     def bind(self, ctx, args, args_grad=None, grad_req='write',
              aux_states=None, group2ctx=None, shared_exec=None):
-        """Bind current symbol to get an executor.
+        """Binds the current symbol to get an executor.
+
+        We first declare the computation and then bind it with the data to run.
+        This function returns an executor which provides method `forward()` for evaluation
```

Comment:
How about:

This function returns an executor which provides a `forward()` method for evaluation and a `outputs()` method to get all the results.